### ISSUE TITLE
[Feature Request] Progress at in backupNotificationHandler

### ISSUE THREAD

**Issue by Chaos02** [NeoApplications/Neo-Backup#520]
**Is your feature request related to a problem? Please describe it.**
Big apps might take a long time backing up - might seem stuck.

**Describe the primary solution**
Add Android notification progress bar (like when downloading a file for example) per app,
If possible without percentage readout

**Describe alternative solutions, if any**
Pipe Output of tar/gzip to notification 

**System Information (please complete the following information):**

- Device: Galaxy S7 SM-G930F
- Android Version: 11
- ROM: (AOSP, MIUI, OOS...)  Lineage os 18.1 unofficial build by Ivan_meler on xda
- App Version: 7.0.0

**Comment by machiav3lli** [comment]
Since 8.0.0 early alphas we have an indicator in the notification for each appin an batch action. You can get it from the TG/Matrix community groups if you'd like to test it and givefurther feedback.

**Comment by Reporter1** [comment]
Yep can confirm, im running the alpha and the drop down has a improved progress indicator (apk, prt, med and 2 bod icons not sure what those mean?).
